00003_string_operations.js

```diff
@@ -0,0 +1,97 @@
+// 3. Enter 2 strings and do the following operations
+// concatenate / reverse / reverse concat / mix the letters / split in half / remove all white spaces / add a character after every specific letter /
+
+let string1;
+let string2;
+
+const concatinate = (string1, string2) => {
+    let concatinatedResult = string1.concat(string2);
+    return concatinatedResult
+}
+
+const reverseString = () => {
+    let reverseString1 = string1.split('').reverse().join('');
+    console.log("Reverse of the first string :", reverseString1);
+    let reverseString2 = string2.split('').reverse().join('');
+    console.log("Reverse of the second string :", reverseString2);
+    let reverseConcat = reverseString1.concat(reverseString2);
+    console.log("Reverse concatinated :", reverseConcat);
+}
+
+const stringMixUp = (string1, string2) => {
+    let stringMix = [];
+    if (string1.length > string2.length) {
+        for (let i = 0; i < string1.length; i++) {
+            if (i + 1 > string2.length) {
+                stringMix = stringMix + string1[i];
+            } else {
+                let currentStringValuesofString1AndString2 = string1[i].concat(string2[i]);
+                stringMix[i] = currentStringValuesofString1AndString2;
+            }
+
+        }
+    } else {
+        for (let i = 0; i < string2.length; i++) {
+            if (i + 1 > string1.length) {
+                stringMix = stringMix + string2[i];
+            } else {
+                let currentStringValuesofString1AndString2 = string1[i] + string2[i];
+                stringMix[i] = currentStringValuesofString1AndString2;
+            }
+
+        }
+
+    }
+    let stringMixResult = stringMix.replace(/,/g, "");
+    return stringMixResult
+}
+
+const splitedHalfOfString1 = (string1) => {
+    if (string1.length % 2 == 0) {
+        return string1.slice(0, string1.length / 2)
+    } else return "cannot be devided."
+}
+
+const splitedHalfOfString2 = (string2) => {
+    if (string2.length % 2 == 0) {
+        return string2.slice(0, string2.length / 2)
+    } else return "cannot be devided."
+}
+
+const spaceOfString1 = (string1) => {
+    let spaceRemovedString1 = string1.replace(/ /g, "");
+    return spaceRemovedString1
+}
+
+const spaceOfString2 = (string2) => {
+    let spaceRemovedString2 = string2.replace(/ /g, "");
+    return spaceRemovedString2
+}
+
+const string1WithExtraCharacter = (string1) => {
+    let string1ExtraCharacter = []
+    for (let i = 0; i < string1.length; i++) {
+        string1ExtraCharacter[i] = string1[i].concat("$")
+    }
+    let string1WithExtraCharacterWithoutComma = String(string1ExtraCharacter).replace(/,/g, "");
+    return string1WithExtraCharacterWithoutComma
+}
+
+const string2WithExtraCharacter = (string2) => {
+    let string2ExtraCharacter = []
```

Comment:
Why you name the variable like this inside the  function ? Let's discuss about this

Reply:
Yea. Actually I had removed the waiting-for-review label from this program. When I removed the readLIne and console, I had to change lot of work flows of this program. I will update this program completely. There have been lot of rearrangements.